enroll.go

```diff
@@ -163,21 +164,24 @@ func (r *EnrollmentResponse) ConfirmEmail(ctx context.Context, code string) (*En
 // PhoneOption represents the method by which the code will be sent to your phone
 type PhoneOption string
 
+// PhoneOption constants.
 const (
 	PhoneMethodVoiceCall PhoneOption = "voice"
 	PhoneMethodSMS       PhoneOption = "sms"
 )
 
+// OktaVeriftyOption is a verify option type.
 type OktaVerifyOption string
 
+// OktaVeriftyOption constants.
 const (
 	OktaVerifyOptionQRCode OktaVerifyOption = "qrcode"
 	OktaVerifyOptionEmail  OktaVerifyOption = "email"
 	OktaVerifyOptionSms    OktaVerifyOption = "sms"
 )
 
 // VerifyPhone sends verification code to the provided phone.
-// Your phone number should contain a country code
+// Your phone number should contain a country code.
```

Comment:
nit: We also need to include the `+` sign in the phone number. Can we give an example of a number with that? Something like +11231231234?

Reply:
yes, I'll elaborate there.